TF_Settings_Web/src/Components/Pages/Camera/Masking/handRendering.ts

```diff
@@ -0,0 +1,90 @@
+import { v4 as uuidgen } from 'uuid';
+
+import { Vector } from 'TouchFree/Configuration/ConfigurationTypes';
+import { ConnectionManager } from 'TouchFree/Connection/ConnectionManager';
+import {
+    ActionCode,
+    CommunicationWrapper,
+    HandRenderDataStateRequest,
+} from 'TouchFree/Connection/TouchFreeServiceTypes';
+import { RawFinger, RawHand } from 'TouchFree/TouchFreeToolingTypes';
+
+import { HandSvgCoordinate, HandSvgProps } from 'Components/Controls/HandsSvg';
+
+export const setHandRenderState = (handRenderState: boolean, lens: string): void => {
+    const requestID = uuidgen();
+
+    const content = new HandRenderDataStateRequest(requestID, handRenderState, lens);
+    const request = new CommunicationWrapper(ActionCode.SET_HAND_DATA_STREAM_STATE, content);
+
+    const jsonContent = JSON.stringify(request);
+
+    ConnectionManager.serviceConnection()?.SendMessage(jsonContent, requestID, () => {
+        return true;
+    });
+};
+
+const translateToCoordinate = (coordinate: Vector | undefined) => {
+    if (coordinate === undefined) return new HandSvgCoordinate(-1, -1, 600);
+    return new HandSvgCoordinate(1000 * (1 - coordinate.X * 1) - 100, 1000 * coordinate.Y - 100, coordinate.Z);
+};
+
+const tipJointIndex = 3;
+const knuckleJointIndex = 1;
+
+export const handToSvgData = (hand: RawHand, handIndex: number): HandSvgProps => {
+    const indexFinger = hand.Fingers.find((f: RawFinger) => f.Type == 1);
+    const middleFinger = hand.Fingers.find((f: RawFinger) => f.Type == 2);
+    const ringFinger = hand.Fingers.find((f: RawFinger) => f.Type == 3);
+    const littleFinger = hand.Fingers.find((f: RawFinger) => f.Type == 4);
+    const thumbFinger = hand.Fingers.find((f: RawFinger) => f.Type == 0);
+    const wrist = hand.WristPosition;
+    return {
+        indexTip: translateToCoordinate(indexFinger?.Bones[tipJointIndex]?.NextJoint),
+        indexKnuckle: translateToCoordinate(indexFinger?.Bones[knuckleJointIndex]?.PrevJoint),
+        middleTip: translateToCoordinate(middleFinger?.Bones[tipJointIndex]?.NextJoint),
+        middleKnuckle: translateToCoordinate(middleFinger?.Bones[knuckleJointIndex]?.PrevJoint),
+        ringTip: translateToCoordinate(ringFinger?.Bones[tipJointIndex]?.NextJoint),
+        ringKnuckle: translateToCoordinate(ringFinger?.Bones[knuckleJointIndex]?.PrevJoint),
+        littleTip: translateToCoordinate(littleFinger?.Bones[tipJointIndex]?.NextJoint),
+        littleKnuckle: translateToCoordinate(littleFinger?.Bones[knuckleJointIndex]?.PrevJoint),
+        thumbTip: translateToCoordinate(thumbFinger?.Bones[tipJointIndex]?.NextJoint),
+        thumbKnuckle: translateToCoordinate(thumbFinger?.Bones[knuckleJointIndex]?.PrevJoint),
+        wrist: translateToCoordinate(wrist),
+        primaryHand: hand.CurrentPrimary,
+        dotColor: handIndex ? 'blue' : 'red',
```

Comment:
Hand index being a number but being used as a bool -> could make this an enum or a type?